cmd/create/execution.go

```diff
@@ -0,0 +1,200 @@
+	 iamRoleURN: 'example: arn:aws:iam::12345678:role/defaultrole'
+	 inputs:
+	   sorted_list1:
+	   - 0
+	   sorted_list2:
+	   - 0
+	 kubeServiceAcct: ""
+	 sourceDomain: development
+	 sourceProject: flytectldemo
+	 targetDomain: development
+	 targetProject: flytectldemo
+	 task: core.advanced.run_merge_sort.merge
+	 version: ""
+
+
+The generated file can be modified to change the input values.
+
+.. code-block:: yaml
+
+	 genExecSpecFile: true
+	 iamRoleURN: 'example: arn:aws:iam::12345678:role/defaultrole'
+	 inputs:
+	   sorted_list1:
+	   - 2
+	   - 4
+	   - 6
+	   sorted_list2:
+	   - 1
+	   - 3
+	   - 5
+	 kubeServiceAcct: ""
+	 sourceDomain: development
+	 sourceProject: flytectldemo
+	 targetDomain: development
+	 targetProject: flytectldemo
+	 task: core.advanced.run_merge_sort.merge
+	 version: ""
+
+And then can be passed through the command line.
+
+::
+
+ bin/flytectl create execution --file core.advanced.run_merge_sort.merge.execution_spec.yaml
+
+Usage
+`
+)
+
+//go:generate pflags ExecutionConfig --default-var executionConfig
+
+// ProjectConfig Config hold configuration for project create flags.
+type ExecutionConfig struct {
+	File            string                 `json:"file,omitempty" pflag:",file for the execution params.If not specified defaults to <<workflow/task>_name>.inputs.yaml"`
+	GenExecSpecFile bool                   `json:"genExecSpecFile,omitempty" pflag:",bool flag to indicate the generation of execution spec file."`
+	Workflow        string                 `json:"workflow,omitempty" pflag:",workflow name for which execution needs to be launched.Either an execution can be for task or workflow."`
+	SourceDomain    string                 `json:"sourceDomain" pflag:",domain from where the workflow or task needs to be fetched for creating the execution.This is mandatory if file is not specified."`
+	SourceProject   string                 `json:"sourceProject" pflag:",project from where the workflow or task needs to be fetched for creating the execution.This is mandatory if file is not specified."`
+	TargetDomain    string                 `json:"targetDomain" pflag:",project where execution needs to be created.If not specified sourceDomain would be used."`
+	TargetProject   string                 `json:"targetProject" pflag:",project where execution needs to be created.If not specified sourceProject would be used."`
+	Task            string                 `json:"task,omitempty" pflag:",task name for which execution needs to be launched.Either an execution can be for task or workflow."`
+	KubeServiceAcct string                 `json:"kubeServiceAcct" pflag:",kubernetes service account AuthRole for launching execution."`
+	IamRoleURN      string                 `json:"iamRoleURN" pflag:",iam role URN AuthRole for launching execution."`
+	Version         string                 `json:"version" pflag:",version of the launch plan or task to be fetched for execution.If not specified it would use the latest version."`
```

Comment:
```suggestion
	Version         string                 `json:"version" pflag:",[optional]version of the launch plan or task to be fetched for execution.If not specified it would use the latest version."`
```

Reply:
Since the file has serialized version field always , We should disallow passing this as an override to avoid any mismatch in the generated file task/lp version against the one passed in. Let me know what you think